modules/virt-enable-vm-action-confirmation-web.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * virt/managing-vms/virt-controlling-vm-states.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-enable-vm-action-confirmation-web_{context}"]
+
+= Enabling confirmations of virtual machine actions
+
+The *Stop*, *Restart*, and *Pause* actions can display confirmation dialogs if confirmation is enabled. By default, confirmation is disabled.
+
+.Procedure
+
+. In the *Virtualization* section of the {product-title} web console, navigate to *Overview* -> *Settings* -> *Cluster* -> *General settings*.
+. Toggle the *VirtualMachine actions confirmation* setting to On.
```

Comment:
```suggestion
= Enabling confirmations of virtual machine actions

The *Stop*, *Restart*, and *Pause* actions can display confirmation dialogs if actions confirmation is enabled. By default, actions confirmation is disabled.

.Procedure

. In the *Virtualization* section of the {product-title} web console, navigate to *Overview* -> *Settings* -> *Cluster* -> *General settings*.
. Toggle the *VirtualMachine actions confirmation* setting to On.
```

Reply:
I also don't love it, but that seems to be what it's called :sweat_smile: I don't know how much discretion we have in changing what something is called in the GUI when we document it - I rarely doc GUI-based features